plugins/native_dio_adapter/pubspec.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
 
   dio: ^5.4.0
   cupertino_http: ^1.0.0
-  cronet_http: ^0.4.0
+  cronet_http: ^1.0.0
```

Comment:
The changelog says

> No functional changes

so I guess we can support

`>=0.4.0 < 2.0.0`. What do you think?

Reply:
```suggestion
  cronet_http: '>=0.4.0 <=2.0.0'
```